fury/ui.py

```diff
@@ -3146,24 +3152,32 @@ def __init__(self, labels, padding=1, font_size=18,
         position : (float, float)
             Absolute coordinates (x, y) of the lower-left corner of
             the button of the first option.
+        checked_labels: list(string)
+            List of labels that are checked on setting up.
         """
+
         self.labels = list(reversed(labels))
         self._padding = padding
         self._font_size = font_size
         self.font_family = font_family
+        self.checked_labels = []
+        self.checked_labels.extend(checked_labels)
```

Comment:
This will fail when `checked_labels` is `None` (i.e. the default value).
I suggest setting the default value to `[]`, then do a soft copy like so
`self.checked_labels = list(checked_labels)`

Reply:
Hi @MarcCote, thank you very much for your review. I have changed most of the things according to your suggestion. However, setting default value to [] seems to be quite dangerous. I did some research on google, and found a better to this issue: https://docs.python-guide.org/writing/gotchas/

So I made the change according to the suggestions above. Pls let me know if there are any more issues :)

Reply:
Interesting link, I did not know that and I need to test it. Thank you for this information.

BTW, instead of

```python
if checked_labels is None:
    self.checked_labels = []
```

you can do it a more pythonic way:

```python
self.checked_labels = checked_labels or []
```

Reply:
Yes, you are right to be cautious of any mutable objects as an argument's default value. That's why the `list(...)` was important in my example.

On a similar note, @skoudoro what you proposed
> self.checked_labels = checked_labels or []
would modify the original checked_labels list since it is passed as a reference.

Reply:
Thank you for the feedback @MarcCote.  Indeed, you are right, I will update this on a new PR.